File: example/python_java_c/prim_raw.c
#include<stdio.h>
#include<math.h>

int main(){
	unsigned int i,j,k = 0;
	for(i = 2;i < (1 << 16);i++)
		for(j = 2;j <= sqrt(i);j++)
			if(i % j == 0){
				k++;
				break;
			}
	printf("%d\n",k);
	return 0;
}

File: example/python_java_c/prim_raw.java
public class prim_raw{
	public static void main(String[] args){
		int i,j,k = 0,l;
		for(i = 2;i < (1 << 16);i++){
			for(j = 2;j <= (int)Math.sqrt(i);j++)
				if(i % j == 0){
					k++;
					break;
				}
		}
		System.out.println(k);
	}
}

